### DISCUSSION TITLE
Backup and Restore Topics

### DISCUSSION THREAD

**Discussion by themkarimi** [strimzi/strimzi-kafka-operator#7835] [Q&A]
Hi,

I am trying to find a solution for backing up and restoring Kafka Topics, I tried using Velero but I wasn't able to achieve any results, is there any way recommended by Strimzi or have you tried any other solutions for this?

**Comment by Veljen** [comment]
@scholzj  do we have any suggestion to have backup and restore (kafka topics, kafka broker, zookeeper etc) ?   Mirrormaker we are using for DR .

**Reply by scholzj** [reply]
I do not have any suggestions really. Mirror maker is for sure one of the options. But it is quite expensive as you have to run two Kafka clusters.
